packages/plugins/plugin-firebase/package.json

```diff
@@ -45,13 +45,11 @@
   },
   "homepage": "https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-firebase#readme",
   "peerDependencies": {
-    "@segment/analytics-react-native": "*"
-  },
-  "dependencies": {
-    "@react-native-firebase/analytics": "^15.4.0",
-    "@react-native-firebase/app": "^15.4.0"
+    "@segment/analytics-react-native": "*",
+    "@react-native-firebase/app": "^17.x"
   },
   "devDependencies": {
+    "@react-native-firebase/analytics": "^17.3.2",
```

Comment:
This should be also in the `peerDependencies` right? People still need to install it. This will only install it for devs

Reply:
oh, when i do that I run into the same problem I did with the messaging one (ie can't find module)

Reply:
Happens if you leave it twice? As in both as a 'peerDependency' and a 'devDependency'?